style.css

```diff
@@ -1,38 +1,42 @@
-/*-------------------mobile version--------------*/
+/* -------------------mobile version-------------- */
 
 .body {
-  display: grid;
-  grid-template-columns: auto;
+  margin: 0%;
   height: 338.125rem;
   width: 23.438rem;
-  background-color: #E5E5E5;
+  background-color: #e5e5e5;
 }
 
 .header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  background-color: #FFFFFF;
+  background-color: #fff;
   height: 3rem;
   width: 23.438rem;
```

Comment:
- [ ] Good job implementing more than required for the milestone, however, kindly make sure your project is responsive, at least between 375-768 of width for the mobile version :recycle:  as requested [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=The%20website%20should,instead%20of%20pixels) in the project requirements.

![Screenshot from 2023-02-14 19-28-16](https://user-images.githubusercontent.com/97834160/218814233-2a6b12b9-051b-4f25-accb-f78a99039d43.png)

Reply:
please can you explain to me on how to go about on this.